pkg/internal/cnsoperator/cnsfilevolumeclient/cnsfilevolumeclient.go

```diff
@@ -91,6 +94,50 @@ func GetFileVolumeClientInstance(ctx context.Context) (FileVolumeClient, error)
 	return fileVolumeClientInstance, nil
 }
 
+// GetClientVmsFromIPList returns the list of client vms associated with a given External IP address and CnsFileVolumeClient instance
+// Returns an error if any operations fails.
```

Comment:
Can you add the format of `fileVolumeName` that callers need to follow to the function comment? And also specify that the function returns an empty list if the instance doesnt exist OR if the input IP address is not present in this instance.

Reply:
Done